superset-frontend/src/views/CRUD/data/database/DatabaseModal/index.tsx

```diff
@@ -428,6 +429,7 @@ const DatabaseModal: FunctionComponent<DatabaseModalProps> = ({
   onHide,
   show,
   databaseId,
+  setDBEngine,
```

Comment:
I wonder if this property name is optimal because it sounds a bit like an operation instead of a React-style declarative declaration. Can we delete `set` part of that name?

Reply:
agree with this 👍

Reply:
Good point, this sounds mode like a hook for setting state than a value.